django_mailjet/backends.py

```diff
@@ -109,16 +109,15 @@ def _build_standart_message_dict(self, message):
         msg_dict['FromEmail'] = from_email
         msg_dict['FromName'] = from_name
 
-        # msg_dict['To'] = message.to
-        msg_dict['Recipients'] = self._parse_recipients(message, message.to)
+        msg_dict['To'] = ', '.join([sanitize_address(addr, message.encoding) for addr in message.to])
```

Comment:
I don't think changing `Recipients` to `To` is a good idea...

From Mailjet docs...

```
Important: Recipients and To have a different behaviors. The recipients listed in To
will recieve a common message showing every other recipients and carbon copies
recipients. The recipients listed in Recipients will each recieve an seperate message
without showing all the other recipients. 
```